src/commonMain/kotlin/software/momento/kotlin/sdk/config/Configurations.kt

```diff
@@ -0,0 +1,99 @@
+package software.momento.kotlin.sdk.config
+
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.seconds
+
+/**
+ * Prebuilt configurations for different environments.
+ */
+public object Configurations {
+
+    /**
+     * Provides defaults suitable for a medium-to-high-latency dev environment. Permissive timeouts
+     * and relaxed latency and throughput targets.
+     */
+    public object Laptop {
+        /**
+         * Provides the latest recommended configuration for a dev environment.
+         *
+         * <p>This configuration may change in future releases to take advantage of improvements we
+         * identify for default configurations.
+         *
+         * @return the latest Laptop configuration
+         */
+        public val Latest: Configuration = Configuration(
```

Comment:
nit: is this the idiomatic casing for a public property? Asking because I genuinely don't remember :)

Reply:
You're right about the property casing. It should be lower case. I'll update them.